utils/rpms/packaging/Makefile_distro_vars.mk

```diff
@@ -22,19 +22,10 @@ DOCKER   := podman
 SPECTOOL := spectool
 # a Fedora-based mock builder
 # derive the the values of:
-# VERSION_ID (i.e. 7)
-# DISTRO_ID (i.e. el7)
-# DISTRO_BASE (i.e. EL_7)
+# VERSION_ID (i.e. 8)
+# DISTRO_ID (i.e. el8)
+# DISTRO_BASE (i.e. EL_8)
```

Comment:
All changes in `utils/rpms/packaging/` need to be made and validated in [daos-stack/packaging](https://github.com/daos-stack/packaging) first and then that needs to be copied here (this should actually be a submodule but we cannot (yet) use submodules due to Jenkins bad handling of them -- there is a task on the pile to work out those issues).  But more importantly, since we are still supporting CentOS 7 on prior releases, we can't really remove CentOS 7 building from [daos-stack/packaging](https://github.com/daos-stack/packaging).